internals/scripts/extract-intl.js

```diff
@@ -94,44 +92,34 @@ for (const locale of locales) {
   }
 }
 
-/* push `react-intl` plugin to the existing plugins that are already configured in `package.json`
-   Example:
-   ```
-  "babel": {
-    "plugins": [
-      ["transform-object-rest-spread", { "useBuiltIns": true }]
-    ],
-    "presets": [
-      "env",
-      "react"
-    ]
-  }
-  ```
-*/
-plugins.push(['react-intl']);
-
-const extractFromFile = fileName => {
-  return readFile(fileName)
+const extractFromFile = fileName =>
+  readFile(fileName)
     .then(code => {
       // Use babel plugin to extract instances where react-intl is used
-      const { metadata: result } = transform(code, { presets, plugins });
+      let messages = [];
+      try {
+        const output = transformSync(code, { presets, plugins });
```

Comment:
In my code I'm using `await transform(code, { presets, plugins, filename: fileName });` note the `await` instead of sync and the additional `filename` that prevents some warnings

Reply:
`filename` made it into the new version of the script merged in #2350 ;)